src/App.js

```diff
@@ -1,22 +1,33 @@
 import Pagination from "./common/Pagination";
 import Section from "./common/Section";
-import MovieList from "./feautures/MovieList";
+import MovieList from "./features/MovieList";
+import ActorTile from "./features/actors/ActorTile";
 import Header from "./common/Header";
-import { HashRouter } from "react-router-dom";
+import { Routes, Route } from "react-router-dom";
+
 
 function App() {
   return (
-    <HashRouter>
+    <>
```

Comment:
We need to wrap the App in HashRouter

Reply:
How about wrapping in index.js file?
![Uploading obraz.png…]()

Reply:
How abou wrapping the app in index.js file?
![obraz](https://user-images.githubusercontent.com/118211320/221414088-c1482bd3-16a6-4cc7-8107-4af044d9a31c.png)